docs/sources/installation/mac.md

```diff
@@ -62,24 +64,7 @@ brew services start grafana
 
 Default login and password `admin`/ `admin`
 
-
```

Comment:
All removed content was moved to configuration.md. If it is generic rather than Mac-specific, then we should remove the "Mac"